### PR TITLE
SW-5335 Updated date range format to be locale agnostic

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#2611** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2611?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/utils/dateFormatter.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const getDateTimeDisplayValue = (timestamp: number): string => {
 };
 
 /**
- * Returns <Date>–<Date> (eg. 7/1-7/15)
+ * Returns <ISO Date> – <ISO Date> (eg. 2023/07/01 - 2023/07/15)
```
Comment (joshbuchea) [inline src/utils/dateFormatter.ts:50]: I'm seeing MM/DD/YYYY – MM/DD/YYYY in Terraware web right now

**Comment by joshbuchea** [comment]
Thanks for this update @tommylau523 💚

File: src/utils/dateFormatter.ts (hunk 1 above)
Comment (tommylau523) [reply src/utils/dateFormatter.ts:50]: Sorry this is reactive to locale currently. I had pushed a fix to remove locale dependencies entirely.

**Review by nickgraz — APPROVED** [review]
Looks good, can you update the title?

**Pull request merged**